libraries/pom.xml

```diff
@@ -206,6 +206,12 @@
             <artifactId>serenity-core</artifactId>
             <version>${serenity.version}</version>
             <scope>test</scope>
+            <exclusions>
```

Comment:
@pivovarit important change

Reply:
this particular version of serenity-core was including a particular version of asciidoctorj which was including jruby-1.7.26 a very old release that contains joda.time classes which were conflicting with our joda-time included as dependency.